php/Terminus/Collections/SshKeys.php

```diff
@@ -2,6 +2,8 @@
 
 namespace Terminus\Collections;
 
+use Terminus\Exceptions\TerminusException;
```

Comment:
@ronan Just trying to run ssh-keys:add I think, and when I gave it invalid input (none at all), it tried to throw this exception which was not found.